File: joao pedro/Problema0335/Problema0335.js
const prompt = require("prompt-sync")({ sigint: true });

var frase,palavra,palavras=[],vezes;
palavra="aula";
vezes=0
frase=(prompt("Digite uma frase que sera detectado o numero de vezes que 'Aula' aparece: "));
frase1=frase.toLowerCase();
 palavras= frase1.split(" ");
for(i=0;i<palavras.length;i++){
    if(palavras[i]==palavra){
        vezes++;
    }
}
console.log("Vezes que 'aula' aparece: "+vezes);

File: joao pedro/Problema0335/Problema0335.java

import java.util.Scanner;
public class Problema0335 {
    public static void main(String[] args) {
        Scanner input=new Scanner(System.in);
        String frase;
        
        String palavra="aula";
        int vezes=0;
        System.out.println("Digite uma frase que sera detectado o numero de vezes que 'Aula' aparece: ");
        frase=input.nextLine();
        String [] palavras= frase.split(" ");
        for(int i=0;i<palavras.length;i++){
            if(palavras[i].equalsIgnoreCase(palavra)){
                vezes++;
            }
        }
        System.out.println("Vezes que 'aula' aparece: "+vezes);
    }

}
